cdap-cli/src/main/java/co/cask/cdap/cli/command/artifact/DescribeArtifactCommand.java

```diff
@@ -87,8 +88,9 @@ public String getPattern() {
 
   @Override
   public String getDescription() {
-    return String.format("Shows information about %s. If no scope is given, the user scope will be used. " +
-      "Includes information about application and plugin classes contained in the artifact.",
+    return String.format("Shows information about %s. If no scope is given, the artifact is looked up in all " +
```

Comment:
Well, if the user has the same artifact name in both system and user scope, then the one in system scope always win, unless the user specify the scope. I think it's good to have clarity (at least that was what I wonder when I read the description... maybe no user will care in the way I do).